sdk/tables/Azure.Data.Tables/tests/TableClientQueryExpressionTests.cs

```diff
@@ -213,9 +214,36 @@ public void TestDictionaryTableEntityFilterExpressions(string expectedFilter, Ex
 
         [TestCaseSource(nameof(UnSupportedTableItemExpressionTestCases))]
         [Test]
+        [NonParallelizable]
         public void TestTableItemFilterExpressionsUnsupported(Expression<Func<TableEntity, bool>> expression)
         {
             Assert.Throws<NotSupportedException>(() => TableClient.CreateQueryFilter(expression));
         }
+
+        [TestCaseSource(nameof(UnSupportedTableItemExpressionTestCases))]
```

Comment:
```suggestion
        [TestCaseSource(nameof(UnsupportedTableItemExpressionTestCases))]
```

big nit and likely out of scope for what you're doing.   Feel free to "won't fix" me, but it was driving me nuts.  😄